app/src/main/java/com/github/se/eventradar/viewmodel/EventsOverviewViewModel.kt

```diff
@@ -40,38 +46,63 @@ constructor(
     }
   }
 
-  fun getUpcomingEvents(uid: String) {
+  fun getUpcomingEvents() {
```

Comment:
Is there a particular reason you removed uid here?

Reply:
This change is part of a broader update where user identification is handled directly within the repository layer, specifically using the new getCurrentUserId() method from the IUserRepository interface.